README.md

```diff
@@ -35,7 +35,7 @@ import {applyStyle} from 'ol-mapbox-style';
 import VectorTileLayer from 'ol/layer/VectorTile.js'
 
 const layer = new VectorTileLayer({declutter: true});
-applyStyle(layer, 'mapbox://styles/mapbox/bright-v9', {accessToken: 'YOUR_MAPBOX_TOKEN'});
```

Comment:
Oh, alright. I made the PR because the `applyStyle` method is not working anymore with `(layer, [url], config)` as parameters. It works with `(layer, src, ...otherParams)` just as the `apply` method with map instead of layer.

Reply:
Ok, can you provide an example of what does not work anymore please?

Reply:
Ah, I get it now. Will create a pull request to make the advertised API work again.